Dockerfile

```diff
@@ -32,11 +32,10 @@ RUN set -x \
  && /fossology/install/scripts/php-conf-fix.sh --overwrite \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
-RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
 
 ADD . .
 RUN chmod +x /fossology/docker-entrypoint.sh
+RUN make composer_download && make composer_install
```

Comment:
Well, I agree with you !! That's why at first, when we starting talking about that, my recommendation was to say: 
integrate composer in your tree. Create a script to update the version if needed. But put it under git, and at least all these stuff disappear, and the update of composer is managed separately, fully outside of build/install.

This is tru because this is not provided by default by distributions on which you install and provide FOSSology. So yes, *my* way of doing it would be to create a small script to downlaod (or update) composer, and put it under your $TOP/bin dire, and then call it from there, knowing it will be there. WDYT ? I can cook a patch for this instead.